io.atom.electron.Sdk.json.in

```diff
@@ -0,0 +1,268 @@
+            ]
+        },
+        {
+            "name": "udev",
+            "config-opts": [ "--disable-hwdb", "--disable-logging", "--disable-gudev",
+                             "--disable-introspection", "--disable-keymap", "--disable-mtd_probe"],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://ftp.kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.xz",
+                    "sha256": "6e7c7330a8f7a1d118e33338941381faa005759921caa241535d6930bb12140f"
+                }
+            ]
+        },
+        {
+            "name": "dbus-glib",
+            "config-opts": [ "--disable-static", "--disable-gtk-doc" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://dbus.freedesktop.org/releases/dbus-glib/dbus-glib-0.106.tar.gz",
+                    "sha256": "b38952706dcf68bad9c302999ef0f420b8cf1a2428227123f0ac4764b689c046"
+
+                }
+            ]
+        },
+        {
+            "name": "gconf",
+            "config-opts": [ "--disable-static", "--disable-gtk-doc", "--disable-orbit",
+                             "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/GConf/3.2/GConf-3.2.6.tar.xz",
+                    "sha256": "1912b91803ab09a5eed34d364bf09fe3a2a9c96751fde03a4e0cfa51a04d784c"
+                }
+            ]
+        },
+        {
+            "name": "pango",
+            "config-opts": [ "--disable-gtk-doc", "--disable-man", "--disable-introspection" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/pango/1.40/pango-1.40.1.tar.xz",
+                    "sha256": "e27af54172c72b3ac6be53c9a4c67053e16c905e02addcf3a603ceb2005c1a40"
+                }
+            ]
+        },
+        {
+            "name": "atk",
+            "config-opts": [ "--disable-gtk-doc", "--disable-introspection", "--disable-man" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/atk/2.20/atk-2.20.0.tar.xz",
+                    "sha256": "493a50f6c4a025f588d380a551ec277e070b28a82e63ef8e3c06b3ee7c1238f0"
+                }
+            ]
+        },
+        {
+            "name": "gtk2",
+            "config-opts": ["--disable-gtk-doc", "--disable-introspection", "--disable-man",
+                            "--with-xinput=xfree" ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/gtk+/2.24/gtk+-2.24.30.tar.xz",
+                    "sha256": "0d15cec3b6d55c60eac205b1f3ba81a1ed4eadd9d0f8e7c508bc7065d0c4ca50"
+                }
+            ]
+        },
```

Comment:
Also need `gnome-themes-standard` like this, because that will get you the GTK2 theme.
https://git.gnome.org/browse/gnome-sdk-images/tree/org.gnome.Sdk.json.in#n556